lib/tasks/import.thor

```diff
@@ -68,8 +90,18 @@ class Import
         @record ||= ImportRecord.create(time_started: DateTime.current)
       end
 
-      def importers(sources = options["source"])
-        sources.map { |s| SOURCE_IMPORTERS.fetch(s, nil) }.compact.uniq
+      def file_import_class_to_client(import_class)
+        return SftpClient.for_x2 if import_class.in?(X2_IMPORTERS)
```

Comment:
Is there any meaningful difference between creating one X2 client and passing it in to each of the X2 importers, as opposed to creating a new X2 client for each importer?